src/caffe/util/sampler.cpp

```diff
@@ -96,11 +96,10 @@ void SampleBBox(const Sampler& sampler, NormalizedBBox* sampled_bbox) {
   CHECK_GT(sampler.min_aspect_ratio(), 0.);
   CHECK_LT(sampler.max_aspect_ratio(), FLT_MAX);
   float aspect_ratio;
-  float min_aspect_ratio = std::max<float>(sampler.min_aspect_ratio(),
-                                           std::pow(scale, 2.));
-  float max_aspect_ratio = std::min<float>(sampler.max_aspect_ratio(),
-                                           1 / std::pow(scale, 2.));
-  caffe_rng_uniform(1, min_aspect_ratio, max_aspect_ratio, &aspect_ratio);
+  caffe_rng_uniform(1, sampler.min_aspect_ratio(), sampler.max_aspect_ratio(), &aspect_ratio);
```

Comment:
Thanks for fixing this. Can you fix the lint error by moving `&aspect_ratio` to a new line? You can try to `make lint` to make sure there isn't any lint error. Thanks!